WordPress/src/main/java/org/wordpress/android/ui/prefs/accountsettings/AccountSettingsViewModel.kt

```diff
@@ -315,6 +317,19 @@ class AccountSettingsViewModel @Inject constructor(
         _accountClosureUiState.value = Dismissed
     }
 
+    fun closeAccount() {
+        (accountClosureUiState.value as? Default)?.let { uiState ->
+            _accountClosureUiState.value = uiState.copy(isPending = true)
+
+            launch {
+                delay(3000)
+                (accountClosureUiState.value as? Default)?.let { uiState ->
+                    _accountClosureUiState.value = uiState.copy(isPending = false)
+                }
```

Comment:
np: I have the feeling this can be replaced…
```kt
(accountClosureUiState.value as? Default)?.let { uiState ->
    _accountClosureUiState.value = uiState.copy(isPending = false)
}
// ↑ replace with ↓
_accountClosureUiState.value = uiState.copy(isPending = false)
```

Reply:
Oh, you are right, that extra check is not needed. I pushed an update and will merge.